src/cli.js

```diff
@@ -1,3 +1,6 @@
+// Disable warnings about template strings in this file, as we do lots of
+// string concatenation so options/help in the CLI app display nicely.
+// jscs:disable requireTemplateStrings
```

Comment:
I have this here as the ESLint complains about multi-line strings and I'm not sure how others feel about them. I think, especially in this file, they're cleaner. Contrast:

```js
   .option('target-maxversion', {
     describe: "JSON string to override the package's " +
       'targetapp_maxVersion for validation. The JSON object ' +
       'should be a dict of versions keyed by application ' +
       "GUID. For example, setting a package's max Firefox " +
       'version to 5.*: ' +
       '{"{ec8030f7-c20a-464f-9b0e-13a3a9e97384}": "5.*"} ',
     type: 'string',
   })
```

with:

```js
.option('target-maxversion', {
  describe: "JSON string to override the package's \
             targetapp_maxVersion for validation. The JSON object \
             should be a dict of versions keyed by application \
             GUID. For example, setting a package's max Firefox \
             version to 5.*: \
             {\"{ec8030f7-c20a-464f-9b0e-13a3a9e97384}\": \"5.*\"}"
  type: 'string',
})

I find the second cleaner and easier to write. I can add in allowing multiline if you like it (everything still built and ran using them, obviously).
```